tests/frameworks/test_huggingface.py

```diff
@@ -101,6 +101,7 @@ def args():
         evaluation_strategy="epoch",
         num_train_epochs=2,
         save_strategy="epoch",
+        report_to="none",
```

Comment:
Docs are [here](https://huggingface.co/docs/transformers/main/en/main_classes/trainer#transformers.TrainingArguments.set_logging.report_to). Having this set to the default of `"all"` seems to create a second instance of the `DVCLiveCallback` which causes duplicate entries to be logged in all of the metric files.

@dberenbaum is this something we need to be aware of for #740?

Reply:
[total_flos](https://huggingface.co/docs/transformers/en/main_classes/callback#transformers.TrainerState.total_flos) was also added as a metric before `report_to` is set to `"none"` (original failure)